src/contentContainer/index.js

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 import { compose } from 'recompose';
 
 const styles = {
-  app: {
+  contentContainer: {
```

Comment:
This component is a `ContentContainer`, and this style is only really applied to the `container` part

Reply:
Fixed 👍